controller/api/models.py

```diff
@@ -672,6 +672,14 @@ def save(self, **kwargs):
                 setattr(self, attr, data)
         except Config.DoesNotExist:
             pass
+
+        for k, v in getattr(self, 'cpu').copy().viewitems():
+            self.cpu[k] = int(v)
```

Comment:
Note that this brittle code may be something that the [serializers](http://www.django-rest-framework.org/api-guide/serializers/) can handle better. See https://github.com/deis/deis/blob/master/controller/api/serializers.py#L26